update to support ccloud blog post (#6)
mongodb-university/kafka-edu: images/stockgenmongo/README.md

# Stock Generation app for MongoDB

> This application will randomly create ficticuous company names, stock symbols and sample data.  These data will be perpetually insert data into MongoDB.  

Reviewer: ```suggestion
This application will create sample data -- randomly generated company names, stock symbols, and other information -- and continuously insert it into MongoDB. 
```
